ingestion-lambda/src/handler.ts

```diff
@@ -134,6 +135,12 @@ export const main = async (event: SQSEvent): Promise<SQSBatchResponse> => {
 							console.warn(
 								`A record with the provided external_id (messageId: ${messageId}) already exists. No new data was inserted to prevent duplication.`,
 							);
+						} else {
+							console.log({
```

Comment:
I was very tempted to make an abstraction along the lines of

```
function logMetric({ eventType: EventType, payload: CorrespondingPayload }) {
...
}
```

But currently this is the only application so I thought it best to avoid premature abstraction for now..